hack/test-cmd.sh

```diff
@@ -67,6 +68,7 @@ export ETCD_PEER_PORT=${ETCD_PEER_PORT:-27001}
 setup_env_vars
 export SUDO=''
 mkdir -p "${ETCD_DATA_DIR}" "${VOLUME_DIR}" "${FAKE_HOME_DIR}" "${MASTER_CONFIG_DIR}" "${NODE_CONFIG_DIR}" "${LOG_DIR}"
+reset_tmp_dir
```

Comment:
It would be nice if we were passing arguments to these functions instead of declaring globals that are used inside of them. Not sure if this is the pull to do that in.